microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -106,11 +116,11 @@ When you are done, learn how to work with the simulator through [Apple Developer
 > [!NOTE]
 > To avoid having to open Xcode every time you want to use the iOS Simulator, open <!--Xcode, or iOS Simulator?-->it, right-click the iOS Simulator icon in your dock, and then select **Keep in Dock**.  Now just click the icon whenever you need it.
 
-### Microsoft Edge (EdgeHTML)
 
-![modern.IE VM](../media/device-mode-modern-ie-vm.msft.png)
+<!-- ------------------------------ -->
+#### Microsoft Edge (EdgeHTML)
 
-Microsoft Edge (EdgeHTML) Virtual Machines (VMs) enable you to access different versions of EdgeHTML and Internet Explorer on your computer through VirtualBox (or VMWare).  Select a [virtual machine on the download page](https://developer.microsoft.com/microsoft-edge/tools/vms).
+If you need to test your website or app with Microsoft browsers and don't have the necessary versions of Windows to do so, you can use BrowserStack, which supports testing of many combinations of Microsoft browsers and operating systems both past and present.  Testing of Microsoft Edge is free on BrowserStack.  For more information, see [Microsoft Edge Browser Testing](https://www.browserstack.com/test-on-microsoft-edge-browser) at BrowserStack.
```

Comment:
This looks good to me, but I would like to have @zoherghadyali's opinion about whether we should mention IE mode here. What do you think Zoher?

Reply:
I'd actually discourage actively talking about IE mode in this doc. We want to be very explicit that IE mode is **not** an emulation feature and we do not plan to support using IE mode to emulate IE as an older browser. IE mode's purpose is for business-critical legacy apps that need IE to function and are prohibitive in cost to upgrade.

Reply:
added versions sentence